groclient/experimental.py

```diff
@@ -3,31 +3,65 @@
 
 
 class Experimental(GroClient):
-    """
-    Experimental class to consume v2prime data.
-    """
+    """Experimental class to consume v2prime data."""
 
     def get_data_points(self, **selections):
-        """
-        Get all the data points for a given selection from /v2prime/data.
+        """Get all the data points for a given selection from /v2prime/data.
+
+        This function is a part of an experimental class and subject to change.
+
+        Parameters
+        ----------
+        metric_id : integer
+            How something is measured. e.g. "Export Value" or "Area Harvested"
+        item_ids : integer or list of integers
+            What is being measured. e.g. "Corn" or "Rainfall"
+        region_ids : integer or list of integers
+            Where something is being measured e.g. "United States Corn Belt" or "China"
+        partner_region_ids : integer or list of integers, optional
+            partner_region refers to an interaction between two regions, like trade or
+            transportation. For example, for an Export metric, the "region" would be the exporter
+            and the "partner_region" would be the importer. For most series, this can be excluded
+            or set to 0 ("World") by default.
+        source_id : integer
+        frequency_id : integer
+        unit_id : integer, optional
+        start_date : string, optional
+            All points with end dates equal to or after this date
+        end_date : string, optional
```

Comment:
looks like we should remove `equal` for both start & end date

see https://github.com/gro-intelligence/gro/blob/main/storage/data_access/src/core_dbs/queries/serving_v2_queries.rs#L347

Reply:
Great catch thanks.